clayui.com/src/components/Editor/index.js

```diff
@@ -78,12 +108,64 @@ const Editor = ({
 						title={collapseCode ? 'Expand' : 'Collapse'}
 					/>
 
-					{!collapseCode && (
+					{Array.isArray(code) ? (
 						<>
-							{imports && <LiveEditor disabled value={imports} />}
+							<ClayTabs modern>
+								{code.map((snippet, index) => (
+									<ClayTabs.Item
+										active={activeTabKeyValue === index}
+										innerProps={{
+											'aria-controls': `tabpanel-${snippet.name}`,
+										}}
+										key={snippet.name}
+										onClick={() =>
+											setActiveTabKeyValue(index)
+										}
+									>
+										{snippet.name}
+									</ClayTabs.Item>
+								))}
+							</ClayTabs>
 
-							<LiveEditor />
+							{!collapseCode && (
+								<ClayTabs.Content
+									activeIndex={activeTabKeyValue}
+									fade
+								>
+									{code.map((snippet) => (
+										<ClayTabs.TabPane
+											aria-labelledby={`tab-${snippet.name}`}
+											key={snippet.name}
+										>
+											{snippet === reactSnippet &&
```

Comment:
Note that we shouldn't do equality values with objects since they can easily be flakey if they are mutated